package.json

```diff
@@ -3,19 +3,22 @@
   "version": "2.1.0",
   "description": "Simple in-memory vinyl file store",
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
```

Comment:
maybe `tsc && mocha`? Or is `mocha` catching on automatically?

Reply:
Or we can also do with `pretest: tsc`

Reply:
I thought it did, but I was wrong. Pushed a commit with `tsc && mocha`. I don't quite like that it'll affect the filesystem, but we're gitignoring dist so it's not a big deal. Jest supports this, but I'm not sure that alone warrants a rewrite from mocha to jest (especially not in this PR).

Reply:
Yeah, most of my OSS are under jest, but this project is a bit old, so I didn't update that part 😂 

Anyway, that's fine! Might be easy to replace mocha with jest since they basically have the same public APIs.